mobile/lib/features/wallet/application/wallet_service.dart

```diff
@@ -26,7 +28,11 @@ class WalletService {
       FLog.info(text: "Successfully created invoice.");
       return invoice;
     } catch (error) {
-      FLog.error(text: "Error: $error", exception: error);
+      if (error is FfiException && error.message.contains("cannot provide required liquidity")) {
```

Comment:
that's not easy to pull off in FRB and not essential. I have left a TODO that eventually it would be nice to do it, but it works just as good, especially that there's a note to not change the comment in the other place.

Reply:
FfiExceptions do have an error code. It's better to use the error code instead of matching the error message.

https://pub.dev/documentation/flutter_rust_bridge/latest/flutter_rust_bridge/FfiException-class.html